inc/Functions/CheckValidEmail.php

```diff
@@ -11,7 +11,7 @@
  */
 function CheckValidEmail($email)
 {
-    global $cfg,$db;
+    global $cfg, $db, $database;
```

Comment:
Function uses neither `$db nor `$database`
```suggestion
    global $cfg
```